usaspending_api/awards/models.py

```diff
@@ -102,9 +103,9 @@ class Award(DataSourceTrackedModel):
         ('L', 'Loan'),
     )
 
-    type = models.CharField(max_length=5, choices=AWARD_TYPES, verbose_name="Award Type")
+    type = models.CharField(max_length=5, choices=AWARD_TYPES, verbose_name="Award Type", null=True)
```

Comment:
@afrasier Makes sense. Let's let @alexjajabah and @playmakervi weigh in on the use of `NULL` here.